tools/duplicate-diff.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+A="modules/file-system/filestore/scripts/mount.sh"
```

Comment:
There may be easier ways to maintain this. For instance, if we had a file called duplicates.csv and then have all duplicates listed there, one per line, then this script could just look at that and process for each line. It would make the code simpler, and updates simpler as well.. 
The file would look like this: 
modules/file-system/filestore/scripts/mount.sh, modules/file-system/pre-existing-network-storage/scripts/mount.sh
community/modules/file-system/nfs-server/scripts/install-nfs-client.sh,modules/file-system/filestore/scripts/install-nfs-client.sh,modules/file-system/pre-existing-network-storage/scripts/install-nfs-client.sh

Reply:
I am moving this change into #648. I will move this comment/discussion into that PR.